lib/services/audio_service_helper.dart

```diff
@@ -79,6 +79,7 @@ class AudioServiceHelper {
     try {
       items = await _jellyfinApiHelper.getInstantMix(item);
       if (items != null) {
+        items.insert(0, item);
```

Comment:
I'd suggest checking for duplicates so 10.10 jellyfin is not regressed

```suggestion
        if (items[0].ItemId != item.ItemId) items.insert(0, item);
```

UPD: Oops, already mentioned here xD

Reply:
also, don't commit this suggestion, it should be `item.id` in both cases :)